docs/mindsdb_sql/knowledge-bases.mdx

```diff
@@ -1047,6 +1031,140 @@ Here is the output:
+
+This test table stores test data commonly in form of questions and answers. Its content depends on the `version` parameter defined below.
+
+Users can provide their own test data or have the test data generated by the `EVALUATE KNOWLEDGE_BASE` command, which is performed when setting the `generate_data` parameter defined below.
+
+### `version`
+
+This parameter defines the version of the evaluator. It is an optional parameter with default value of `doc_id`.
+
+* `version = 'doc_id'`
+    The evaluator checks whether the document ID returned by the knowledge base matched the expected document ID as defined in the test table.
+
+* `version = 'llm_relevancy'`
+    The evaluator uses a language model to rank and evaluate responses from the knowledge base.
+
+### `generate_data`
+
+This is an optional parameter used to configure the tests data generation, which is saved into the table defined in the `test_table` parameter.
+
+Available values are as follows:
+
+* A dictionary containing the following values:
+    * `from_sql` defines the SQL query that fetches the test data. For example, `'from_sql': 'SELECT content FROM my_datasource.my_table'`.
+    * `count` defines the size of the test dataset. For example, `'count': 100`. Its default value is 20.
+
+* A value of `true`, such as `generate_data = true`, which implies that default values for `from_sql` and `count` will be used.
+
+### `evaluate`
+
+This is an optional parameter that defines whether to evaluate the knowledge base.
+
+Users can opt for setting it to false, `evaluate = false`, in order to generate test data into the test table without running the evaluator.
+
+### `llm`
+
+This is an optional parameter that defines a language model to be used for evaluations, if `version` is set to `llm_relevancy`.
+
+By default it uses the [`reranking_model` defined with the knowledge base](/mindsdb_sql/knowledge-bases#reranking-model).
+
+Users can define it with the `EVALUATE KNOWLEDGE_BASE` command in the same manner.
+
+```sql
+EVALUATE KNOWLEDGE_BASE my_kb
+USING
+    ...
+    llm = {
+        "provider": "azure_openai",
+        "model_name" : "gpt-4o",
+        "api_key": "sk-abc123",
+        "base_url": "https://ai-6689.openai.azure.com/",
+        "api_version": "2024-02-01",
+        "method": "multi-class"
+    },
+    ...
+```
+
+### `save_to`
+
+This is an optional parameter that stores the name of the table from one of the data sources connected to MindsDB. For example, `save_to = my_datasource.my_result_table` defines a table named `my_result_table` from the data source named `my_datasource`.
+
+This table is used to save the evaluation results.
+
+By default, evaluation results are returned after executing the `EVALUATE KNOWLEDGE_BASE` statement.
+
+### Evaluation Results
+
+When using `version = 'doc_id'`, the following columns are included in the evaluation results:
```

Comment:
I mixed up the names of versions in[ PR description](https://github.com/mindsdb/mindsdb/pull/10914). Corrected now. parameters for `doc_id` should be for `llm_relevancy`